src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -166,8 +175,41 @@ internal struct curl_version_info_data
             private int versionNum;
             private unsafe char *host;
             internal int features;
+            internal IntPtr ssl_version;
+        }
+
+        internal static partial class CurlSslBackend
```

Comment:
Ok. The current code is meant to be a sort of workaround for an issue at libcurl's end. 
I will move the function to a different file.  May be I will move it back here, when it is fixed here (then this is the right place for the associated types involved).